felix/bpf/tc/attach.go

```diff
@@ -59,6 +59,7 @@ type AttachPoint struct {
 	PSNATEnd             uint16
 	IPv6Enabled          bool
 	MapSizes             map[string]uint32
+	BTFEnabled           bool
```

Comment:
Is this really attach point's feature? Does not seems so? What if we moved `SupportsBTF()` into `bpf/bpfutils` and call it in `init()` and set it as a global. We could just import `bpfutils` here and would not need to plumb through the whole configuration.

CTLB and other hooks we have could also use that information easily then.